src/tools/rbd/ArgumentTypes.h

```diff
@@ -81,7 +81,9 @@ static const std::string PRETTY_FORMAT("pretty-format");
 static const std::string VERBOSE("verbose");
 static const std::string NO_ERROR("no-error");
 
-static const std::string DELAY("delay");
+static const std::string EXPIRES_AT("expires-at");
```

Comment:
Nit: since these arguments are really only used by ```Trash.cc```, we can move them all there while making these changes.